recipe/meta.yaml

```diff
@@ -57,8 +59,14 @@ test:
 about:
```

Comment:
You could add a test command using pdm like `- pdm --python`

You also miss an entrypoint in the top build section (see: https://github.com/pdm-project/pdm/blob/2.2.1/pyproject.toml#L54)

Reply:
`- pdm --python` didn't seem to work, but I did add `- pdm --version`

Reply:
That works too